content/docs/platform/concepts/environments.mdx

```diff
@@ -4,42 +4,63 @@ description: 'Understanding and managing environments in Novu'
 icon: 'GitFork'
 ---
 
-Novu uses environments to separate your development and production workflows. When you create an account, you'll get two default environments: `Development` and `Production`.
+Novu uses environments to separate your development and production workflows. When you create an account, you'll get two default environments: **Development** and **Production**.
 
-## Development Environment
+## Development environment
 
 Use the development environment to test new notification workflows, validate changes before deploying to production, and experiment with different configurations.
 
-## Production Environment
+The Development environment is your primary workspace for building and testing. This is the only environment where you should create and modify workflows, layouts, and notification translations.
```

Comment:
@Aviatorscode2 

As customers in the `team` and `enterprise` tiers can create custom environment, so those environments can also act as non production environment. Can you please rephrase this line and remove the word `primary` and `only`